ui/src/components/files/upload_progress_bar/mod.rs

```diff
@@ -205,17 +205,17 @@ pub fn UploadProgressBar<'a>(cx: Scope<'a, Props>) -> Element<'a> {
     }
 
     if *cx.props.are_files_hovering_app.read() {
```

Comment:
To make it cleaner, you could do the following

```rust
if !*cx.props.are_files_hovering_app.read() {
  return None;
}
  
cx.render(rsx!(
            div {
                class: "upload-progress-bar-container-file-count",
                p {
                    id: "upload-file-count",
                    class: "upload-file-count",
                }
            },
))
  
```

Not a blocker though for this